qcodes/dataset/descriptions/rundescriber.py

```diff
@@ -49,32 +49,13 @@ def version(self) -> int:
         return self._version
 
     @property
-    def shapes(self) -> Shapes:
+    def shapes(self) -> Shapes | None:
         return self._shapes
 
     @property
     def interdeps(self) -> InterDependencies_:
         return self._interdeps
 
-    @staticmethod
-    def _verify_interdeps_shape(interdeps: InterDependencies_,
```

Comment:
was this an unused method?

Reply:
Its a private method that is not used anymore